scripts/globalTypes.d.lua

```diff
@@ -7927,7 +7927,6 @@ declare class PluginPolicyService extends Instance
 end
 
 declare class PluginToolbar extends Instance
-	function CreateButton(self, buttonId: string, tooltip: string, iconname: string, text: string?): PluginToolbarButton
 	function CreateButton(self, id: string, toolTip: string, iconAsset: string, text: string?): PluginToolbarButton
```

Comment:
Do we even need this CreateButton override as well?

Reply:
I think the parameter names is an improvement, so may as well keep it
`iconAsset` is clearer that it's an image rather than `iconname`, when I first saw `iconname` in the tooltip I was confused what it did